src/app/metadata-schema-form/metadata-form/metadata-form.component.ts

```diff
@@ -118,11 +118,18 @@ export class MetadataFormComponent implements OnInit {
   }
 
   getFormData() {
-    const formValue = this.metadataForm.formGroup.getRawValue();
-    const formData = this.metadataFormService.cleanFormData(formValue);
-
+    let formValue = this.metadataForm.formGroup.getRawValue();
+    if (Array.isArray(this.config.cleanFields)) {
```

Comment:
Why are you checking if it's an array? What else could it be?

Reply:
```
cleanFields?: boolean | string[];
```

Reply:
The behaviour of clean fields is covered by the new clean fields tests [here](https://github.com/ebi-ait/ingest-ui/blob/48a13fd36d43a00aaaec5da86fb8b93d2329e26d/src/app/metadata-schema-form/metadata-form/metadata-form.component.spec.ts#L109)